config/application.yml.default

```diff
@@ -296,7 +296,7 @@ development:
   hmac_fingerprinter_key_queue: '["11111111111111111111111111111111", "22222222222222222222222222222222"]'
   identity_pki_local_dev: true
   idv_api_enabled_steps: '["password_confirm", "personal_key","personal_key_confirm"]'
-  kantara_2fa_phone_restricted: false
+  kantara_2fa_phone_restricted: true
```

Comment:
Is this value supposed to be flipped at this time?

Reply:
yes this is in development, in particular so its on by default for folks in local dev.